src/components/BookLists.js

```diff
@@ -8,10 +8,10 @@ const BookLists = () => {
     <div>
       {books.map((book) => (
         <Book
-          key={book.id}
-          id={book.id}
-          title={book.title}
-          author={book.author}
+          key={book[0]}
+          id={book[0]}
+          title={book[1][0].title}
+          author={book[1][0].author}
         />
```

Comment:
- [ ] As required by the project requirements. Kindly import useEffect (from react) and useDispatch from react-redux into your `BookList` comonent, then dispatch your fetchBook from inside useEffect.

<img width="691" alt="image" src="https://user-images.githubusercontent.com/52934128/230051276-69969d01-fcfb-4293-9a8c-d24797b06b92.png">